waspc/cli/Wasp/Cli/Terminal.hs

```diff
@@ -1,9 +1,40 @@
 module Wasp.Cli.Terminal
   ( title,
+    asWaspMessage,
+    asWaspStartMessage,
+    asWaspSuccessMessage,
+    asWaspFailureMessage,
   )
 where
 
 import qualified Wasp.Util.Terminal as Term
 
 title :: String -> String
 title = Term.applyStyles [Term.Bold]
+
+asWaspMessage :: String -> String
+asWaspMessage = waspMessage ""
+
+asWaspStartMessage :: String -> String
+asWaspStartMessage = waspMessage "🐝"
+
+asWaspSuccessMessage :: String -> String
+asWaspSuccessMessage = waspMessage "✅"
+
+asWaspFailureMessage :: String -> String
+-- Add a bit more padding on errors for more pronounced
+-- visibility and better display of any following error context.
+asWaspFailureMessage str = concat ["\n", waspMessage "❌" errorStr, "\n"]
+  where
+    errorStr = "[Error] " ++ str
+
+waspMessage :: String -> String -> String
```

Comment:
it was a bit hard for me to get an idea of what this one does without looking at the implementation, esp when compared to `asWaspXXX` functions. maybe it could be named `waspMessageWithEmoji`? But not a big deal since it is an internal helper function, just thinking out loud.

Reply:
Yep good call, I think that can be cleaned up some to be more descriptive, even if internal only. 👍🏻 Will do